src/ralph/scan/plugins/ssh_cisco_asa.py

```diff
@@ -104,12 +104,24 @@ def scan_address(ip_address, **kwargs):
     ssh = _connect_ssh(ip_address, SSH_USER, SSH_PASS)
     try:
         lines = ssh.asa_command(
-            "show version | grep (^Hardware|Boot microcode|^Serial|address is)"
+            "show version | grep (^Hardware|Boot microcode|address is)"
         )
+        lines2 = ssh.asa_command("show inventory")
     finally:
         ssh.close()
+
+    inventory = [item for item in lines2 if item != u'\r']
```

Comment:
there is `from __future__ import unicode_literals` at the top of the file, so you don't have to use `u` before string:
`u'\r'` -> `'\r'`